external/storm-kafka-client/pom.xml

```diff
@@ -130,7 +130,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.kafka</groupId>
-            <artifactId>kafka_2.11</artifactId>
+            <artifactId>kafka_2.13</artifactId>
```

Comment:
same remark as previous, shouldn't it be 3.6.0 ?

Reply:
Ah okay, that's confusing ;)

Reply:
Yes. I also had to look twice.